### ISSUE TITLE
Projects - Move to be 3rd Page

### ISSUE THREAD

**Issue by jamescd18** [Northeastern-Electric-Racing/NER-PM-Dashboard#71]
Move the order of the page tabs at the top of the website so that the "Projects" tab appears as the third tab. Very similar to #55.

**Comment by MeerHamza1421** [comment]
Can you please explain this issue a bit more

**Comment by eshwaribhide** [comment]
Maybe for consistency sake, everything under "Projects Page Functions" in JavaScript.html can also be moved right below dashboard page functions (assuming at this point dashboard page functions will be 2nd tab)

**Comment by jamescd18** [comment]
Yes, the order of the code chunks in the `JavaScript.html` files would be nice to have match the order of the web app's tabs, just as they match on the home page bullet list.
